web/app.js

```diff
@@ -890,15 +887,29 @@ const PDFViewerApplication = {
   },
 
   /**
-   * Opens PDF document specified by URL or array with additional arguments.
-   * @param {string|TypedArray|ArrayBuffer} file - PDF location or binary data.
+   * Opens PDF document specified by URL or other additional arguments.
    * @param {Object} [args] - Additional arguments for the getDocument call,
```

Comment:
This line doesn't really describe what the function expects, which is mostly a pre-existing issue but now that `file` is removed I feel it became more important. `args` should probably not be optional anymore, but the contents of the object should also be detailed using e.g. a typedef like we have elsewhere to describe parameter objects. That would make it clear for the reader which values are legal to pass, which I guess from the rest of the PR are the `GetDocumentParameters` plus some other values like `originalUrl` (and maybe more).

Reply:
That's a fair point, I've update the JSDoc to hopefully improve things (without being too specific to avoid it becoming outdated too easily).